src/lv_gpu/lv_gpu_stm32_dma2d.c

```diff
@@ -214,15 +214,6 @@ void lv_gpu_stm32_dma2d_blend(lv_color_t * buf, lv_coord_t buf_w, const lv_color
  *   STATIC FUNCTIONS
  **********************/
 
-static void invalidate_cache(void)
```

Comment:
If we delete this function the user needs to take care to register a cache cleaner in the display driver.

What about keeping this function and call the custom cleaner if present else  `SCB_CleanInvalidateDCache`?

Reply:
That would work, except that there'll be linking errors if `SCB_CleanInvalidateDCache` isn't available in the application, which means people would have to implement a fake stub to satisfy the linker.

Reply:
`SCB_CleanInvalidateDCache` is a CMSIS function, so I think we can assume that it's there.

If not - for whatever reasons - users still have a workaround.

Reply:
Okay; I've updated it to work the way you suggested.